core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -0,0 +1,140 @@
+package org.bitcoins.core.protocol.tlv
+
+import org.bitcoins.core.number.UInt16
+import org.bitcoins.core.protocol.BigSizeUInt
+import org.bitcoins.crypto.{Factory, NetworkElement}
+import scodec.bits.ByteVector
+
+sealed trait TLV extends NetworkElement {
+  def tpe: BigSizeUInt
```

Comment:
type

Reply:
reserved word, in scala the convention is to use `tpe` for this reason

Reply:
Ah yeah, forgot

Reply:
&#96;type&#96;

Reply:
or tlvType